bitwarden-rs-alpine.json

```diff
@@ -0,0 +1,36 @@
+{
+    "Bitwarden-rs": {
+        "description": "Bitwarden-rs on Alpine",
```

Comment:
I would develop this a little bit more, so that users unfamiliar with Bitwarden know what it is used for. A single sentence should suffice.